### PR TITLE
[webdriver]: element cloned into iframe

### PR DESCRIPTION
**Pull request by MoLow**
Added a new test to verify webdriver can access & serialize elements transferred in between iframes
following this [bug](https://bugs.webkit.org/show_bug.cgi?id=228270)

### CODE REVIEW

File: webdriver/tests/switch_to_frame/switch_webelement.py [hunk 1]
```diff
@@ -82,3 +92,22 @@ def test_frame_id_webelement_no_frame_element(session, inline):
 
     response = switch_to_frame(session, no_frame)
     assert_error(response, "no such frame")
+
+def test_frame_id_webelement_cloned_into_iframe(session, inline, iframe):
+    session.url = inline(iframe("<body><p>hello world</p></body>"))
+
+    response = execute_script(session, """
```
Comment (jgraham) [inline webdriver/tests/switch_to_frame/switch_webelement.py:99]: You can just do `session.execute_script` instead of adding the helper.

**Review by jgraham — CHANGES_REQUESTED** [review]
Looks good, just a small piece of cleanup.

File: webdriver/tests/switch_to_frame/switch_webelement.py [hunk 2]
```diff
@@ -82,3 +83,20 @@ def test_frame_id_webelement_no_frame_element(session, inline):
...
     assert_error(response, "no such frame")
+
+def test_frame_id_webelement_cloned_into_iframe(session, inline, iframe):
+    session.url = inline(iframe("<body><p>hello world</p></body>"))
+
+    session.execute_script("""
+            const iframe = document.getElementsByTagName('iframe')[0];
+            const div = document.createElement('div');
+            div.innerHTML = 'I am a div created in top window and appended into the iframe';
+            iframe.contentWindow.document.body.appendChild(div);
```
Comment (whimboo) [inline webdriver/tests/switch_to_frame/switch_webelement.py:94]:
```suggestion
    session.execute_script("""
        const iframe = document.getElementsByTagName('iframe')[0];
        const div = document.createElement('div');
        div.innerHTML = 'I am a div created in top window and appended into the iframe';
        iframe.contentWindow.document.body.appendChild(div);
```

File: webdriver/tests/switch_to_frame/switch_webelement.py [hunk 3]
```diff
@@ -82,3 +83,20 @@ def test_frame_id_webelement_no_frame_element(session, inline):
...
+    """)
+
+    frame = session.find.css("iframe", all=False)
+    response = switch_to_frame(session, frame)
+    assert_success(response)
+
+    element = session.find.css("div", all=False)
+    assert element.text == "I am a div created in top window and appended into the iframe"
```
Comment (whimboo) [inline webdriver/tests/switch_to_frame/switch_webelement.py:102]:
```suggestion
    assert element.text == "I am a div created in top window and appended into the iframe"

```

**Comment by whimboo** [comment]
Looks fine to me. I'll leave the final  review for @jgraham including the merge. Thanks for adding this testcase!

**Review by jgraham — APPROVED** [review]

**Comment by MoLow** [comment]
thanks, @jgraham - who should handle the merging of the releated WebKit issue? https://bugs.webkit.org/show_bug.cgi?id=228270

**Comment by whimboo** [comment]
That's up to anyone as CC'ed on this bug. Maybe @gsnedders can help? But any disussion for that should happen on this mentioned bug.

**Pull request merged**